tests/cli/test_correct_scraper.py

```diff
@@ -15,8 +15,8 @@
 
 tests = {
     'release': {
-        'args': ['-v', '23.0.1'],
-        'fname': 'firefox-23.0.1.en-US.linux64.tar.bz2',
+        'args': ['-v', '23.0.1', '-p', 'win32'],
```

Comment:
Yes, this looks good - this is just testing a specific os now, but this is the same kind of test, no regression here I think. Yeah you can do a better coverage later (my personal preference is [pytest and its parametrize](http://pytest.org/latest/parametrize.html#pytest-mark-parametrize-parametrizing-test-functions) functionality for this kind of things.